lib/pages/plan_details_page.dart

```diff
@@ -84,45 +92,42 @@ class _CaregiverPlanDetailsPageState extends State<CaregiverPlanDetailsPage> {
 		);
 	}
 
-	Segment _getAdditionalTasksSegment({String title, String noElementsMessage, List<UITask> tasks}) {
-		return Segment(
-			title: title,
-			noElementsMessage: '$_pageKey.content.noTasks',
-			elements: <Widget>[
-				for (var task in tasks)
-					Padding(
-						padding: EdgeInsets.symmetric(horizontal: AppBoxProperties.screenEdgePadding),
-						child: TaskCard(
-							task: TaskFormModel(
-								key: ValueKey(DateTime.now()),
-								title: task.name,
-								timer: task.timer,
-								optional: true,
-								pointsValue: task.points != null ? task.points.quantity : null,
-								pointCurrency:  task.points != null ? UICurrency(type: task.points.type, title: task.points.title) : null
-							)
-						)
-					)
-			]
-		);
-	}
 
-	Widget _getCardHeader(UIPlan plan) {
+	Widget _getCardHeader(UIPlan plan, Map<Mongo.ObjectId, String> children, BuildContext context) {
+  	int i = 0;
+// ignore: close_sinks
+		var authenticationBloc = context.bloc<AuthenticationBloc>();
+		var currentUser = authenticationBloc.state.user;
+
+
 		return AppHeader.widget(
 			title: '$_pageKey.header.title',
 			appHeaderWidget: ItemCard(
 				title: plan.name,
 				subtitle: plan.description(context),
+				//TODO: Open child page with click on chip
 				chips: <Widget>[
-					AttributeChip.withIcon(
-						content: AppLocales.of(context).translate('page.caregiverSection.plans.content.tasks', {'NUM_TASKS': plan.taskCount}),
-						color: Colors.indigo,
-						icon: Icons.layers
-					)
+					if (currentUser.role == UserRole.child)
+						AttributeChip.withIcon(
+							content: AppLocales.of(context).translate('page.caregiverSection.plans.content.tasks', {'NUM_TASKS': plan.taskCount}),
+							color: Colors.indigo,
+							icon: Icons.layers
+						)
+					else if (children.isNotEmpty)
```

Comment:
Put brackets around the following if and else to make clear that both cases are for caregiver